commonContent.json

```diff
@@ -185,14 +190,19 @@
       },
       "webchat": {
         "title": "Sgwrsio dros y we",
-        "desc": "Gallwch gael cymorth trwy sgwrsio ag asiant ar-lein.",
+        "desc": "Cael help drwy siarad ag asiant ar-lein.",
         "textChatDown": "Nid yw’r adnodd sgwrsio dros y we ar gael ar hyn o bryd.\n Rhowch gynnig arall arni yn hwyrach ymlaen neu ffoniwch ni ar y rhif isod os yw’ch ymholiad yn un brys.",
-        "textChatWithAnAgent": "Cychwyn sgwrs dros y we (agor mewn ffenestr newydd)",
+        "textChatWithAnAgent": "Dechrau sgwrsio dros y We (agor mewn ffenestr newydd)",
         "textNoAgentsAvailable": "Nid oes asiant ar gael ar hyn o bryd; rhowch gynnig arall arni yn hwyrach ymlaen.",
         "textAllAgentsBusy": "Mae ein hasiantau sgwrsio dros y we i gyd yn brysur ar hyn o bryd yn helpu pobl eraill.\n Rhowch gynnig arall arni yn hwyrach ymlaen neu cysylltwch â ni drwy un o’r dulliau isod.",
         "textChatClosed" : "Mae’r gwasanaeth sgwrsio dros y we nawr ar gau. \n Gallwch ddefnyddio’r gwasanaeth hwn o ddydd Llun i ddydd Gwener, 8.30am - 5pm, neu cysylltwch â ni drwy un o’r dulliau isod.",
         "textChatAlreadyOpen": "Mae ffenestr sgwrsio dros y we ar agor yn barod.",
-        "textAdditional": "Dydd Llun i ddydd Gwener, 8.30am – 5pm"
+        "textAdditional": "Dydd Llun i ddydd Gwener, 8.30am – 5pm",
+        "localeUrl": "/assets/locale/cy-gb",
+        "language": "Welsh",
+        "chatTitle": "Sgwrsio dros y we GLlTEF",
+        "pageName": "Apelio yn erbyn penderfyniad ynghylch budd-dal",
+        "jsFile": "welsh"
```

Comment:
I think we can use one language property (probably lower-case), and use titlecase filter in the nunjucks view to make it appear as Welsh/English